test/e2e/common/networking.go

```diff
@@ -126,5 +126,27 @@ var _ = ginkgo.Describe("[sig-network] Networking", func() {
 				}
 			}
 		})
+
+		// Once basic tests checking for the sctp module not to be loaded are implemented, this
+		// needs to be marked as [Disruptive]
+		ginkgo.It("should function for intra-pod communication: sctp [LinuxOnly][Disruptive]", func() {
```

Comment:
Hm... that comment should have been updated (everywhere) once we _did_ mark the tests `[Disruptive]`:

    // [Disruptive] because it conflicts with tests that call CheckSCTPModuleLoadedOnNodes

Reply:
done